ktor-client/ktor-client-tests/jvm/test/io/ktor/client/tests/ExceptionsJvmTest.kt

```diff
@@ -62,4 +71,22 @@ class ExceptionsJvmTest {
             }
         }
     }
+
+    @Test
+    fun testContentNegotioationMediaType() = runBlocking {
+        val client = HttpClient(Apache) {
+            install(ContentNegotiation) {
+                json()
+            }
+        }
+
+        val response = client.post("https://httpbin.org/post") {
```

Comment:
better to avoid 3rd party services and real network in tests

Reply:
Thanks for the review! I've removed this service from the test.